tests/emu/run_emu.py

```
@@ -0,0 +1,254 @@
+            NFAIL += 1
+
+    return 0, NFAIL, NELMS
+
+def main():
+    if len(sys.argv) < 2:
+        print(f"usage: {sys.argv[0]} <file.hsaco>", file=sys.stderr)
+        return 1
+
+    print("=== IPL checks (smoke tests) ===")
+    TPAS1 = ipl1()
+    TPAS2 = ipl2()
```

Ruff (0.15.1)

[error] Local variable `TPAS2` is assigned to but never used

Remove assignment to unused variable `TPAS2`

(F841)